### ISSUE TITLE
Manta Comics: Inverted chapter list

### ISSUE THREAD

**Issue by Sylvan-Knight** [tachiyomiorg/tachiyomi-extensions#11123]
### Source information

Manta Comics 1.2.2 

### Steps to reproduce

1. Select a comic (e.g. The Demon Queen Has a Death Wish)
2. Select a chapter that is not titled Chapter 1
3. Scroll through the chapter into the following chapter

### Expected behavior

Expectation 1: After Step 1, the chairs should load in reverse chronological order with the newest chapter at the top and the oldest chapter at the bottom

Expectation 2: After finishing a chapter it should go to the next chapter numerically.

### Actual behavior

Actual 1: The chapters load in reverse, even though the date information is correct.

Actual 2: After completing a chapter, it goes to the previous chapter numerically. Finishing chapter 2 takes your to chapter 1 instead of 3

### Tachiyomi version

0.13.1

### Android version

Android 12

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
